tests/scripts/thread-cert/node.py

```diff
@@ -1155,6 +1156,17 @@ def remove_service(self, enterpriseNumber, serviceData):
         self.send_command(cmd)
         self._expect('Done')
 
+    def __getOmrAddress(self):
```

Comment:
There should be only one OMR address. So, just return one or None?

Reply:
Yeah, there should be at most one. But to detect if there is at most one, we need to return a list.